contracts/LimitsChecker.sol

```diff
@@ -0,0 +1,289 @@
+// SPDX-FileCopyrightText: 2022 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.4;
+
+import "./libraries/EVMScriptCreator.sol";
+import "./interfaces/IBokkyPooBahsDateTimeContract.sol";
+
+import "OpenZeppelin/openzeppelin-contracts@4.3.2/contracts/access/AccessControl.sol";
```

Comment:
As long as that's a single OZ contract we're using, let's just copy-paste it to our sources & audit? Will get verification _significantly_ easier.

Reply:
It would require to remove the brownie dep and ensure that every other ET contract could be compiled

Reply:
done